docs/builder.rst

```diff
@@ -63,9 +72,12 @@ By default, the manifest builder returns a dict representing the manifest. To re
    >>> assert isinstance(built_package, Package)
 
 
+To validate a manifest
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
 
-To validate the built manifest
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+   validate()
```

Comment:
These should probably be contextualized within a `built(...)` for these code examples to be the most helpful.

Reply:
Yup, it's a couple lines below - I thought that just providing the "ingredient" for how to do something at the beginning of each section might be helpful for fast lookup rather than digging through the example below

Reply:
I was actually suggesting something as simple as:

```python
build(
    ...,
    validate(),
)
```

Just to show it is intended to be used within the `build` API.

Reply:
Ahh, gotcha cool will update in following pr